step-release-vis/src/app/components/environment/environment_test.ts

```diff
@@ -340,17 +340,21 @@ describe('EnvironmentComponent', () => {
     const svg = fixture.debugElement.query(
       By.css(`#${component.environment.name}-svg`)
     );
-    svg.triggerEventHandler('click', {pageX: 100, pageY: 100});
+    click(100, 100, svg);
     fixture.detectChanges();
-
     expect(component.tooltip.clickOn).toBeTrue();
 
-    svg.triggerEventHandler('click', {pageX: 100, pageY: 100});
+    click(100, 100, svg);
     fixture.detectChanges();
 
     expect(component.tooltip.clickOn).toBeFalse();
   });
 
+  function click(x: number, y: number, element: DebugElement): void {
```

Comment:
Do you want to use this to test the drag as well? If yes, give a better name. If not, you can hardcode the x, y values.